draft-sheffer-oauth-rfc8725bis.md

```diff
@@ -276,16 +283,14 @@ attacker gets hold of such a token  {{Langkemper}}.
 
 
 
-## Incorrect Composition of Encryption and Signature {#incorrect-composition-of-encryption-and-signature}
+## Incorrect Use and Composition of Encryption and Signature {#incorrect-composition-of-encryption-and-signature}
 
+Most authentication use cases only require a simple signed JWT as their token. However verifiers don't always check that the received JWT is a signed JWS as opposed to an encrypted JWE structure. This can result in vulnerabilities, in particular when the verifier's asymmetric key includes the private key, and when the verifier's library does not distinguish between successful decryption and successful signature validation {{CVE-2023-51774}}.
```

Comment:
```suggestion
Most authentication use cases only require a simple signed JWT as their token. However verifiers don't always check that the received JWT is a signed JWS as opposed to an encrypted JWE structure. This can result in vulnerabilities when the verifier's library does not distinguish between successful decryption and successful signature validation {{CVE-2023-51774}}.
```

Delete the ambiguous phrase

Reply:
Please apply my suggestion deleting the ambiguous phrase, then I will approve.

Reply:
I'm also fine with Mike's phrasing ...